packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md

```diff
@@ -520,10 +502,10 @@ the location of the stack in the construct tree (specifically, moving the Stack
 into a Stage).
 
 
-| Since | Default | Recommended |
+| Since | Unset behaves like | Recommended value |
 | ----- | ----- | ----- |
 | 1.58.0 | `false` | `true` |
-| 2.0.0 | `true` | `true` |
+| 2.0.0 | `false` | `true` |
```

Comment:
Is this updating documentation to reflect the correct value, or is there a behaviour change?

I don't see a behavioural change in the code but just checking anyway.